packages/fxa-auth-server/test/remote/account_signin_verification_tests.js

```diff
@@ -331,6 +331,10 @@ describe('remote account signin verification', function () {
       .then((c) => {
         client2 = c;
       })
+      .then(() => {
+        // Clears inbox of new signin email
+        return server.mailbox.waitForEmail(email);
```

Comment:
If you don't do this what happens? Does it mess up the proceeding 'waitForCode' call?

Reply:
Yea it would since an extra email is now sent.